lib/app/dataMixin.js

```diff
@@ -47,13 +47,13 @@ export default {
       return targetLang || defaultLang || {}
     },
     $title () {
-      return this.$localeConfig.title || this.$site.title || ''
+      return this.$page.frontmatter.title || this.$localeConfig.title || this.$site.title || ''
     },
     $description () {
-      return this.$localeConfig.description || this.$site.description || ''
+      return this.$page.frontmatter.title || this.$localeConfig.description || this.$site.description || ''
```

Comment:
Shouldn't this be `this.$page.frontmatter.description`?

Reply:
Cool, typo. thanks, fixed at 89b8386